### PR TITLE
FastAPI: First steps. Trying to implement a simple Library Service

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    return crud.create_author(db=db, author=author)
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
+    authors = crud.get_authors(db, skip=skip, limit=limit)
+    return authors
+
+
+@app.get("/authors/{author_id}", response_model=schemas.Author)
```
Comment (vasylhnatiuk) [inline main.py]: it's generally a good practice for endpoints to conclude with a trailing slash (/).

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
Provide return type annotations for all functions

File: crud.py
Comment (YukihiroSM) [inline crud.py:1]: Add return types for these functions as well, please :)

**Review by YukihiroSM — CHANGES_REQUESTED** [review]

**Review by NikOneZ1 — APPROVED** [review]
Good Job!

**Pull request closed without merging**